Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -257,6 +257,8 @@
     <Compile Include="$(MSBuildThisFileDirectory)Effects.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)FailImageSource.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)GestureBubblingTests.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Github1461.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)InputTransparentInheritance.cs" />
```

Comment:
@rmarinho  `InputTransparentInheritance` shouldn't have been part of the PR. I guess it came with the merge you did?